br/pkg/storage/parse_test.go

```diff
@@ -69,15 +69,15 @@ func TestCreateStorage(t *testing.T) {
 	require.Equal(t, "TestKey", s3.SseKmsKeyId)
 
 	// special character in access keys
```

Comment:
need to keep the "special character `+`" https://github.com/pingcap/br/pull/614

Reply:
I got it👍, I am discussing with securiy team